app/helpers/application_helper.rb

```diff
@@ -6,8 +6,8 @@ def knowledge_base_url
     'http://openstax.force.com/support/articles/FAQ/Can-t-log-in-to-your-OpenStax-account'
   end
 
-  def openstax_url
-    'https://openstax.org'
+  def accounts_url
+    main_app.root_path
```

Comment:
Isn't this the same as doing `'/'`?

Reply:
Which would point to Accounts (`accounts-dev.openstax.org`, `accounts.openstax.org` in prod, etc.) whereas we want it to point to "osweb" (`cms-dev.openstax.org`, `openstax.org` in prod, etc.)

Reply:
Possibly. I implemented what was used in the old flow based on a conversation with Brittany.

Reply:
Updated to use deployment information for openstax URL